app/pages/settings/profile.jsx

```diff
@@ -0,0 +1,142 @@
+import React from 'react';
+import apiClient from 'panoptes-client/lib/api-client';
+import ImageSelector from '../../components/image-selector';
+import putFile from '../../lib/put-file';
+
+const MAX_AVATAR_SIZE = 65536;
+const MAX_HEADER_SIZE = 256000;
+
+class CustomiseProfile extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      avatar: {},
+      avatarError: null,
+      header: {},
+      headerError: null
+    };
+    this.avatarGet();
+    this.headerGet();
+  }
+
+  avatarGet() {
+    this.props.user.get('avatar')
+    .then(([avatar]) => {
+      this.setState({ avatar });
+    });
+  }
+
+  headerGet() {
+    this.props.user.get('profile_header')
+    .then(([header]) => {
+      this.setState({ header });
+    });
+  }
+
+  handleMediaChange(type, file) {
+    const { user } = this.props;
+    const errorProp = `${type}Error`;
+    this.setState({ [errorProp]: null });
+    apiClient.post(user._getURL(type), { media: { content_type: file.type }})
+    .then(([resource]) => {
+      putFile(resource.src, file, { 'Content-Type': file.type });
+    })
+    .then(() => {
```

Comment:
Why was this block in the original component? What did it do? Can we just delete it?

Reply:
I originally wrote these files and I'm pretty sure I copied what was being done in the lab for the image media uploads. Maybe this commit is a clue? https://github.com/zooniverse/Panoptes-Front-End/commit/627021199705ff433439812cd69e0a7f5f4667bb